### ISSUE TITLE
Set body of request/responce

### ISSUE THREAD

**Issue by rodinv** [pangpanglabs/echoswagger#46]
for marshalling/unmarshalling my service input/output data i use github.com/google/jsonapi ( https://jsonapi.org/ ).
Is there a way to use []Byte(or a string with json data) in methods:
AddResponse (parameter 'schema')
AddParamBody( parameter 'p')
?

**Comment by elvinchan** [comment]
No, because JSON has no sufficient type information corresponding to [OpenAPI 2.0 data type](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types).
For example, for JSON `{"a": 1}`, you don't know is it `int32`, `int64`, `float`, or `double`.
